4 2 timer

IlyaUman opened the pull request:
в качестве аргументов ожидается строка вида 1h 2m 3s или 2m 3s или 3s или пустая строка. Таймер сработает в любом случае не более чем через 5 секунд, сделано специально, чтобы можно было проверить ввод минут и часов и не ждать.

Dokril commented on 4-2-timer/index.js:
```diff
@@ -0,0 +1,26 @@
+
+let secs, mins, hours, timer
+
+if (!process.argv[2]) {
```
Тут лучше получить из аргументов все элементы кроме перых двух а дальше начиная с конца превратить их в значение таймера

IlyaUman committed:
Second task about timer

в качестве аргументов ожидается строка вида 1h 2m 3s или 2m 3s или 3s или пустая строка. Таймер сработает в любом случае не более чем через 5 секунд, сделано специально, чтобы можно было проверить ввод минут и часов и не ждать.

IlyaUman committed:
Second task about timer

в качестве аргументов ожидается строка вида 1h 2m 3s или 2m 3s или 3s или пустая строка. Таймер сработает в любом случае не более чем через 5 секунд, сделано специально, чтобы можно было проверить ввод минут и часов и не ждать.

IlyaUman committed:
Second task about timer

в качестве аргументов ожидается строка вида 1h 2m 3s или 2m 3s или 3s или пустая строка. Таймер сработает в любом случае не более чем через 1 секундe, сделано специально, чтобы можно было проверить ввод минут и часов и не ждать.

IlyaUman committed:
Merge branch 'master' into 4-2-timer
